src/components/styles/form.css

```diff
@@ -0,0 +1,35 @@
+.container {
+  margin: 2rem auto;
+  border-radius: 10px;
+  width: 86%;
+  margin-bottom: 10rem;
+}
+
+.container h2 {
+  margin: 1rem 0.2rem;
+  color: gray;
+}
+
+.container form {
+  padding: 1rem 0.3rem;
+  display: flex;
+  gap: 3rem;
+}
```

Comment:
- [ ] You did a fantastic job implementing the improvements that the previous reviewer had requested 👏. However, I believe you overlooked one modification, which is to make sure the form section is correctly oriented to the left and right with the cards as in the original design. Please ensure to fix that so your design resembles the template as closely as possible. 👌 😊

#### This is what the reviewer pointed out :point_down: 
![Screenshot from 2023-08-26 10-52-59](https://github.com/Mahdi-Niazi/BookStore/assets/89228743/9c1e19e1-085f-4865-8cec-362c36893f92)

#### This is what we still have :point_down: 
![Screenshot from 2023-08-26 10-57-34](https://github.com/Mahdi-Niazi/BookStore/assets/89228743/1deae911-3f99-4c50-9381-3c69e05958fa)

### This is how the template looks like :point_down: 
![Screenshot from 2023-08-26 11-07-29](https://github.com/Mahdi-Niazi/BookStore/assets/89228743/b32844b9-7e06-4084-a177-bf8d56c842cf)

So this :point_down:  is a suggestion to help you fix the issue. 😊

```suggestion
.container {
  margin: 2rem 6.25rem;
  border-radius: 10px;
  margin-bottom: 10rem;
}

.container h2 {
  margin: 1rem 0;
  color: gray;
}

.container form {
  padding: 1rem 0;
  display: flex;
  gap: 3rem;
}
```

What I did here is I removed:
- the `width: 86%;` to the `container` and updated the **margin-left and right** from `auto` to `6.5rem`. 
- updated the **margin-left and right** from `0.2rem` to `0` added to the `h2`
- updated the **margin-left and right** from `0.3rem` to `0` added to the `form`

If you follow their recommendations, the outcome will be as follows :point_down:   which resembles the original design in my opinion.😊

#### Result :point_down: 
![Screenshot from 2023-08-26 11-22-28](https://github.com/Mahdi-Niazi/BookStore/assets/89228743/63239ab2-b4f7-45c4-85a6-5a8e61c1ee59)